frontend/src/core/translation/components/Comment.js

```diff
@@ -0,0 +1,88 @@
+/* @flow */
+
+import * as React from 'react';
+import ReactTimeAgo from 'react-time-ago';
+
+import { WithPlaceables } from 'core/placeable';
+import type { TranslationComment } from 'core/api';
+
+
+type Props = {|
+    comment: TranslationComment,
+|};
+
+
+export default class Comment extends React.Component<Props> {
+    renderUserAvatar() {
+        const { comment } = this.props;
+
+        if (!comment.author_id) {
+            return <img
+                src='/static/img/anon.jpg'
+                alt=''
+                height='44'
+                width='44'
+            />;
```

Comment:
Can  you please clarify what you mean by "factor this out"?  Should it be a functional component and if so under which file?

Reply:
Initially I meant a `renderUserImg()` function, but now that we decided to turn all `renderX` method in components, I'd suggest a component.

Probably in `core/user/components`, so that we can reuse it in `history`'s `Translation.js`, `UserMenu.js`, `FiltersPanel.js` and later on in Team Comments.